schemas/string_interpolation/transforms/telephone_number_link.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/telephone_number_link.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "telephone_number_link": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["telephone_number_link"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "telephone_number": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
```

Comment:
The card says basic validation should be done on the phone number - a simple regex in the schema for numbers, `+` and spaces should be enough.